cockatrice/src/player.cpp

```diff
@@ -2535,6 +2541,9 @@ void Player::processSceneSizeChange(int newPlayerWidth)
 
 void Player::setLastToken(CardInfo *cardInfo)
 {
+    if (cardInfo == nullptr)
```

Comment:
Might want to check the caller of this too to see that it's not using the cardinfo expecting it to be not null